### ISSUE TITLE
methodjit crashes with compiler (gcc) optimizations turned off

### ISSUE THREAD

**Issue by GoogleCodeExporter** [classilla/tenfourfox#150]
```
Aurora 14 currently only starts up when methodjit is disabled. When enabling 
jit during runtime a crash follows immediately.

Attached comes a backtrace from gdb.

I didn't investigate in the reason yet - but maybe there's a chance that 
methodjit could break code that was compiled with PIC turned on (here: position 
independent code)?
```

Original issue reported on code.google.com by `Tobias.N...@gmail.com` on 12 May 2012 at 10:36
- Blocked on: #179
- Merged into: #179

Attachments:
- [backtrace_Aurora14.txt](https://storage.googleapis.com/google-code-attachments/tenfourfox/issue-150/comment-0/backtrace_Aurora14.txt)
